src/components/homepage/zip_export.js

```diff
@@ -10,10 +10,15 @@ const ZipExport = (props) => {
           selected_game: game,
         },
       })
-      .then((data) => {
-        console.log(data);
-        alert("Your data has been logged from this endpoint in the developer conosle.");
-      })
+        .then(({ data }) => {
+          var url = window.URL.createObjectURL(new Blob([data]));
```

Comment:
This section takes the response data as a blob and opens it as a download link. This isn't a great way to handle this if you need backwards compatibility for older browsers. It might also have issues if the content is too big. Typically for bigger files, you might just do a window.open to the url to the download. Then the file that called the window.open would also close that window. By doing this, once they download the file, it will auto close.